src/ServiceControl.Monitoring/Infrastructure/IntervalsStore.cs

```diff
@@ -57,7 +57,7 @@ public Measurement(TimeSpan intervalSize, int numberOfIntervals)
             // ReSharper disable once SuggestBaseTypeForParameter
             public void ReportTimeIntervals(DateTime now, IntervalsBreakdown item)
             {
-                var epoch = GetEpoch(now.Ticks);
+                var epoch = GetEpoch(now.Ticks) - 1;
```

Comment:
Magic const. Can we change it to

```
var currentEpoch = GetEpoch(now.Ticks);
var previousEpoch = currentEpoch - 1;
```

or sth similar to make it more explicit?

Reply:
I've keep the `epoch` instead of `previousEpoch` as it's used inside a look and there it means epoch to be included in reported data.